wordpress-mobile/WordPress-iOS
# Replace KIF with Xcode UI Tests

This refactor removes the KIF library and associated UITests and replaces it with new UITest target available in Xcode 7.

This new version only adds a fraction of the tests because a lot of them where not working or outdated because of Calypso updates on the app.

Going forward new tests should be added as people implement new features on the app, or decided to add tests to feature previously implemented.

How to test:

 * Start Xcode 
 * Select the WordPressUITests schema and run the tests (CMD+U). 

If everything goes right you should see the simulator start and the application working by itself, doing login and testing features in the app. If the tests start failing all over the place, check the following:

 * The hardware keyboard cannot be connected to the simulator if you don’t see the touch keyboard showing up in the simulator, it’s because the hardware keyboard is one. To disable just press SHIFT+CMD+K.

Needs review: @astralbodies
